pkg/component/component.go

```diff
@@ -14,6 +14,8 @@ import (
 	"github.com/redhat-developer/odo/pkg/config"
 	"github.com/redhat-developer/odo/pkg/occlient"
 	"github.com/redhat-developer/odo/pkg/project"
+	"github.com/redhat-developer/odo/pkg/storage"
+	urlpkg "github.com/redhat-developer/odo/pkg/url"
```

Comment:
why add a custom name for this import?

Reply:
@containscafeine  otherwise it collides with `net/url`